### PR TITLE
[BPK-1106] Add divider docs and screenshots to RN card page

### PR DESCRIPTION
**Pull request by matthewdavidson**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -14,21 +15,24 @@
 - bpk-component-banner-alert: 1.3.8 => 1.4.0
   - New `dismissable` configuration, see https://backpack.github.io/components/web/banner-alerts
 
-- react-native-bpk-component-card:
+- react-native-bpk-component-card: 1.1.6 => 1.1.7
```
Comment (matthewdavidson) [inline changelog.md]: This was missed in the last release (also should have been a minor)

Comment (k0nserv) [reply changelog.md]: In my defense there wasn't an entry in the changelog

Comment (matthewdavidson) [reply changelog.md]: No defense required! :)

Comment (matthewdavidson) [reply changelog.md]: In fact you are correct and I erroneously assumed it was released! Fixed.

**Pull request merged**